src/Init/Data/BitVec/Bitblast.lean

```diff
@@ -2154,4 +2154,204 @@ theorem shiftLeft_add_eq_shiftLeft_or {x y : BitVec w} :
     (y <<< x) + x =  (y <<< x) ||| x := by
   rw [BitVec.add_comm, add_shiftLeft_eq_or_shiftLeft, or_comm]
 
+/- ### fast circuit for unsigned overflow detection -/
+/--
+  unsigned parallel prefix `x.uppcRec s = true` iff `x.toNat` is greater or equal than `2 ^ (w - 1 - (s - 1))`
```

Comment:
Should this be a sentence starting with an uppercase letter and terminating with a `.`?

Reply:
It seems this is not a convention in the file, but maybe still double-check.

Reply:
fixed